packages/config/config/devices/0x0258/nas-pd02z-2.json

```
@@ -0,0 +1,261 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NAS_PD02Z-2",
+	"description": "PIR Motion, Temperature & Light Sensor",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0x1036"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Group 1 is lifeline group; all nodes which associated in this group will receive the messages sent by device through lifeline.",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Control",
+			"description": "Group 2 is controlling group, all nodes associated in this group will be controlled through BASIC_SET command by the device when device detects a movement event.",
```

dito, also lets rename the group to be clearer
```suggestion
			"label": "Motion Control",
```